content/home.pml

```diff
@@ -27,6 +27,11 @@
   <div class="news-column news-content">
     <ul>
       <li>
+        <strong>
+          15 Nov 2023: Support for Rust now available on seL4
```

Comment:
In the same vein as my comment about the wording in the first sentence of the news page, this could be "Support for Rust in seL4 userspace now available".